tools/internal_ci/linux/grpc_xds_k8s_run_test.sh

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
```

Comment:
Please rename this file to `grpc_xds_k8s_run_cross_test.sh` or `grpc_xds_k8s_run_xest.sh`. Current name implies it's used by other `xds_k8s` build scripts, which it's not.

Reply:
done